profile_page.php

```diff
@@ -0,0 +1,39 @@
+<?php
+    
+    #include all that you may need
+    #you will get two parameters from the POST REQUEST.(email and verification_code)
+    /* if the verification_code is thesame with the unique_co in the database for the given email, 
+    then the verified column of the Database for that user will be updated to true(1)
+    */
+	
+    session_start();
+	if( $id = $_GET['id'] ?? $_SESSION['userid'] ){
```

Comment:
Even though I have merged this code, I would appreciate it if $_POST is used in the future for any user sensitive data.